src/styles/blocks/month.scss

```diff
@@ -0,0 +1,28 @@
+.month{
+  display: flex;
+  margin: auto;
+  padding: 10px;
+  flex-wrap: wrap;
+  gap: $border-gap;
+  max-width: $one-week;
+  @for $index from 1 through $max-days-mounth  {
+    &__day:nth-child(#{$index}) {
+      cursor: pointer;
+      box-sizing: border-box;
+      border: 1px solid black;
+      background-color: #eee;
+      width: 100px;
+      height: 100px;
+      display: flex;
+      transition: 0.5s;
+      &:hover {
+        background-color: #ffbfbf;
+        transform: translateY(-20px);
+      }
+      &::before {
+        content: "#{$index}";
+        margin: auto;
+      }
+    }
+  }
```

Comment:
if you div's have a class month__day, let's add all common styles just to this class without cycle and use for cycle to add content to ::before